src/components/SelectBox.jsx

```diff
@@ -0,0 +1,51 @@
+import styles from '../styles/selectBox'
+
+import React from 'react'
+import ReactMarkdown from 'react-markdown'
+
+const LANGS_OPTIONS = ['en', 'fr']
+
+const Option = ({t, inputData, optionValue, setValue}) => (
+  <option
+    value={optionValue}
+    selected={setValue === optionValue}
+  >
+    {t(`AccountView.${inputData}.${optionValue}.text`)}
+  </option>
+)
+
+const SelectBox = ({t, inputData, setValue, infosSubmitting, updateInfos}) => (
+  <div className={styles['coz-form']}>
+    <h3>{t(`AccountView.${inputData}.title`)}</h3>
+    <label className={styles['coz-desc']}>
+      {t(`AccountView.${inputData}.label`)}
+    </label>
+    <select
+      name={inputData}
+      onBlur={updateInfos}
+      >
+      {
+        LANGS_OPTIONS.map(function (lang) {
+          return <Option
+            optionValue={lang}
+            inputData={inputData}
+            setValue={setValue}
+            t={t}
+          />
+        })
+      }
+    </select>
+    <p className={styles['coz-desc']}>
+      <ReactMarkdown
```

Comment:
Should we 'officially' use ReactMarkdown to do advanced thing with Transifex? Until now, we used to split the locale each time we want to use html inside. What do you think @m4dz @gregorylegarec and @goldoraf?